src/pages/index.tsx

```diff
@@ -44,30 +45,32 @@ export const actions = [
 
 export const developerLinks = [
   {
-    title: 'uniswap-v3-core',
-    href: 'https://github.com/Uniswap/uniswap-v3-core',
+    title: 'uniswap-v4-core',
+    href: 'https://github.com/Uniswap/v4-core/',
     icon: GitHub,
   },
   {
-    title: 'uniswap-v3-sdk',
-    href: 'https://github.com/Uniswap/uniswap-v3-sdk',
+    title: 'uniswap-v4-periphery',
+    href: 'https://github.com/Uniswap/v4-periphery',
     icon: GitHub,
   },
   {
-    title: 'uniswap-v3-periphery',
-    href: 'https://github.com/Uniswap/uniswap-v3-periphery',
+    title: 'uniswap-v4-sdk',
+    href: 'https://github.com/Uniswap/sdks/tree/main/sdks/v4-sdk',
     icon: GitHub,
   },
   {
     title: 'Deployment addresses',
-    href: 'https://github.com/Uniswap/uniswap-v3-periphery/blob/main/deploys.md',
+    href: '/contracts/v4/deployments',
     icon: GitHub,
   },
+  /*
   {
     title: 'widgets',
     href: 'https://www.npmjs.com/package/@uniswap/widgets',
     icon: Npm,
   },
+  */
```

Comment:
actually im not a big fan of creating deadcode like this so we should just delete this entirely + delete the unused import

```
import Npm from '@site/static/img/npm.svg'
```